test/course.spec.ts

```diff
@@ -38,9 +71,33 @@ describe('CourseService', () => {
       expect(true).toEqual('not implemented');
     });
 
+
+
+    /**
+     * @author Enrico Acquaviva
+     */
     it('ao finalizar o 12 curso, se sim, transformar o usuário em premium', () => {
-      expect(true).toEqual('not implemented');
+      const student = new Student('Enrico', BasicSubscription);
+      const dataStore = new DataStore();
+      dataStore.students.push(student);
+      let i=0;
+      for (i; i<12;i++){
+        const curso = new Course(`Curso${i}`, null);
+        dataStore.courses.push(curso);
+        student.setcompletedCourse(curso);
+      } 
+      const studentService = new StudentService(dataStore);
+      
+      let possuiDozeCursos = studentService.verifyTwelveCourses(student.id);
+
+      if(possuiDozeCursos == true){
+        expect(possuiDozeCursos).toEqual(true);
+      }
```

Comment:
troca para apenas `expect(possuiDozeCursos).toEqual(true);`

```suggestion
      expect(possuiDozeCursos).toEqual(true);
```